README.md

```diff
@@ -169,6 +170,30 @@ window.ai.addEventListener((event: EventType, data: unknown) => {
 })
 ```
 
+**(BETA) Generate 3D Objects**: generate a 3D object using a specified model or the user-preferred model.
+
+```ts
+window.ai.BETA_generate3DObject(
+    input: PromptInput,
+    options?: ThreeDOptions
+): Promise<MediaOutput[]>
+```
+
+The `BETA_generate3DObject` function allows you to generate 3D objects with a defined model and options. The input should be a `PromptInput` `{ prompt : string }`. The options parameter is optional and accepts `ThreeDOptions` customize the media generation request.
+
+Here's an example request:
+
+```javascript
+const result = await window.ai.BETA_generate3DObject(
```

Comment:
nit: better to call this "results". another useful trick: `const [ result ] = await window...`

Reply:
changed to 
<img width="489" alt="image" src="https://github.com/alexanderatallah/window.ai/assets/107221725/a50b32b7-5188-4ca0-a9aa-242f97211a14">